docs/function_guides/allocator.rst

```diff
@@ -95,22 +95,25 @@ routine can be found in ``libE_info``, passed into the allocation function::
 
     libE_info =  {'exit_criteria': dict,               # Criteria for ending routine
                   'elapsed_time': float,               # Time elapsed since start of routine
-                  'manager_kill_canceled_sims': bool,  # True if manager is canceling simulations
+                  'manager_kill_canceled_sims': bool,  # True if manager is to send kills to cancelled simulations
                   'given_count': int,                  # Total number of points given for simulation function evaluation
                   'returned_count': int,               # Total number of points returned from simulation function evaluations
                   'given_back_count': int,             # Total number of evaluated points given back to a generator function
-                  'sim_max_given': bool}               # True if the maximum number of simulations has been performed
+                  'sim_max_given': bool}               # True if the `sim_max` simulations have been given out to workers
 
-Besides the initial allocation function check, the remaining values above are useful
-for very efficient control over how work is distributed to workers, especially for
-authors of persistent generator functions. Generators that construct models based
+
+In most supplied examples, the allocation function will just return once `sim_max_given`
```

Comment:
extra ticks around sim_max_given

Reply:
Don't we usually have ticks in such a place?

Reply:
as in, extra ticks needed in rst, I think? ``sim_max_given`` vs `sim_max_given`. Should've been clearer!

(looks like on github both render the same...)